<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usermanagementuiDlg</class>
 <widget class="QDialog" name="usermanagementuiDlg">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>usermanagementui</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#usermanagementuiDlg
{
	background-image: url(none);
	border-image: url(none);
}
</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,1">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_14">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QWidget" name="widget_left" native="true">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget
{
	background-color: rgb(238, 238, 239);
	border:0px groove gray;
	border-radius:10px;
	padding:2px 4px;
}

QToolButton
{
background-color: rgb(179, 202, 255);
border:0px groove gray;
border-radius:10px;
padding:2px 4px;
	font: 75 18pt &quot;Arial&quot;;
}

QToolButton:checked
{
     background-color: rgb(81, 167, 249);
}

QToolButton:checked:hover
{
     background-color: rgb(81, 167, 249);
}

QToolButton:!checked
{
     background-color: rgb(179, 202, 255);
}

QToolButton:hover 
{
     background-color: rgba(179, 202, 255, 100);
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>128</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_16">
              <item>
               <widget class="QToolButton" name="btnProMode">
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>65</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>65</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>18</pointsize>
                  <weight>9</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>生产模式</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>15</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="btnDebugMode">
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>65</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>65</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>18</pointsize>
                  <weight>9</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>调试模式</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_10">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>15</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="btnEngineerMode">
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>65</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>65</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>18</pointsize>
                  <weight>9</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>工程模式</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_down">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>118</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QWidget" name="widget_right_up" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>301</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16775</width>
         <height>301</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 239);
border:0px;
border-radius:10px;
padding:2px 4px;
</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>90</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>101</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/image/man.png);</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="margin">
             <number>10</number>
            </property>
            <property name="indent">
             <number>10</number>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>用户名：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="editName">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>28</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border:1px groove gray;
border-color: rgb(214, 226, 235);
border-radius:4px;
padding:2px 4px;

</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_7">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>13</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>密码：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="editPassword">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>28</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border:1px groove gray;
border-color: rgb(214, 226, 235);
border-radius:4px;
padding:2px 4px;</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_8">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QToolButton" name="btnLogin">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QToolButton
{
color: rgb(255, 255, 255);
background-color: rgb(255, 160, 40);
border-radius:6px;
padding:2px 4px;
}
QToolButton:hover 
{
     background-color: rgba(255, 160, 40, 100);
	 border:1px groove rgb(255, 160, 40, 100);
}
QPushButton:pressed 
{
     background-color: rgba(255, 160, 40);
	 border:1px groove rgb(255, 160, 40);
 }</string>
               </property>
               <property name="text">
                <string>登录</string>
               </property>
               <property name="shortcut">
                <string>Return</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>128</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="btnCompetence">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QToolButton
{
color: rgb(255, 255, 255);
background-color: rgb(255, 160, 40);
border-radius:6px;
padding:2px 4px;
}
QToolButton:hover 
{
     background-color: rgba(255, 160, 40, 100);
	 border:1px groove rgb(255, 160, 40, 100);
}
QPushButton:pressed 
{
     background-color: rgba(255, 160, 40);
	 border:1px groove rgb(255, 160, 40);
 }</string>
               </property>
               <property name="text">
                <string>权限管理</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnUser">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QToolButton
{
color: rgb(255, 255, 255);
background-color: rgb(255, 160, 40);
border-radius:6px;
padding:2px 4px;
}
QToolButton:hover 
{
     background-color: rgba(255, 160, 40, 100);
	 border:1px groove rgb(255, 160, 40, 100);
}
QPushButton:pressed 
{
     background-color: rgba(255, 160, 40);
	 border:1px groove rgb(255, 160, 40);
 }</string>
               </property>
               <property name="text">
                <string>用户管理</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_right_down" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 239);
border:0px groove gray;
border-radius:10px;
padding:2px 4px;
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="DisplayLogWidget" name="tEditInformationTedit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border:0px groove gray;
border-radius:10px;
padding:2px 4px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayLogWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>displaylogwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
